vrs/marten-construction-script.py

```diff
@@ -0,0 +1,180 @@
+# Daniel Marten 01-20-2023
+# Construct a ~100k Variant Test VCF for GA4GH-VRS Project
+# Reach out to me at: marten@broadinstitute.org
+
+# Import and set up important packages/modules
+import hail as hl
+import random
+from random import sample
+from gnomad.resources.grch38.gnomad import public_release
+
+from gnomad.resources.resource_utils import (
+    MatrixTableResource,
+    VariantDatasetResource,
+    VersionedMatrixTableResource,
+    VersionedVariantDatasetResource,
+)
+
+# Import logging for logs (to replace print statements)
+import logging
```

Comment:
running isort would reorder the imports as follows:

import logging
import random
from random import sample

import hail as hl
from gnomad.resources.grch38.gnomad import public_release
from gnomad.resources.resource_utils import (MatrixTableResource,
                                             VariantDatasetResource,
                                             VersionedMatrixTableResource,
                                             VersionedVariantDatasetResource)